index.html

```diff
@@ -4115,7 +4115,7 @@ <h5>Note regarding the ARIA 1.3 <code>image</code> role.</h5>
 			<rdef>img</rdef>
 			<div class="role-description">
 				<p>A container for a collection of <a>elements</a> that form an image. See synonym <rref>image</rref>.</p>
-				<p>An <code>img</code> can contain captions and descriptive text, as well as multiple image files that when viewed together give the impression of a single image. An <code>img</code> represents a single graphic within a document, whether or not it is formed by a collection of drawing <a>objects</a>. In order for elements with a <a>role</a> of <code>img</code> to be <a>perceivable</a>, authors MUST provide a label using the <pref>aria-label</pref> or <pref>aria-labelledby</pref> attribute.</p>
+				<p>An <code>img</code> can contain captions and descriptive text, as well as multiple image files that when viewed together give the impression of a single image. An <code>img</code> represents a single graphic within a document, whether or not it is formed by a collection of drawing <a>objects</a>. In order for an element with a <a>role</a> of <code>img</code> to be <a>perceivable</a>, authors MUST provide the element with an <a>accessible name</a>. This can be done using the <pref>aria-label</pref> or <pref>aria-labelledby</pref> attribute.</p>
```

Comment:
Do we want to say anything about native host language attributes? (e.g. `title`)

Reply:
I thought about that, but shied away from the idea per my original PR comment.  Specifically didn't want to make someone think that the example we referenced was always the best way to add a name with the host language, and also didn't want to make someone think "oh, i could use the `alt` attribute here."

but if i'm being too wary and others don't think that's an issue, then yes. let's add it :)

Reply:
I agree with @scottaohara on this point.